tests/StarCluster.in

```diff
@@ -0,0 +1,48 @@
+# *****************************************************************
+# Problem size and geometry
+# *****************************************************************
+geometry.prob_lo     =  -10.  -10.  -10.
+geometry.prob_hi     =  10.  10.  10.
+geometry.is_periodic =  0    0    0
+
+# *****************************************************************
+# VERBOSITY
+# *****************************************************************
+amr.v              = 1       # verbosity in Amr
+
+# *****************************************************************
+# Resolution and refinement
+# *****************************************************************
+amr.n_cell          = 128 128 128
+amr.max_level       = 0     # number of levels = max_level + 1
+amr.blocking_factor = 16    # grid size must be divisible by this
+amr.max_grid_size   = 64    # at least 128 for GPUs
+amr.n_error_buf     = 3     # minimum 3 cell buffer around tagged cells
+amr.grid_eff        = 0.7   # default
+
+hydro.reconstruction_order = 3  # PPM
+cfl = 0.2
+max_timesteps = 300
```

Comment:
This simulation is almost certainly hitting `max_timesteps` before it reaches $t = 0.5$.

Reply:
Nope, it takes 290 steps to reach $`t = 0.5`$ haha:

```
Coarse STEP 290 at t = 0.4996641169 (99.93282337%) starts ...
```

Reply:
You must have played around with the max_timesteps in the past ;)

Reply:
Haha, I must have lol.